Login/src/main/java/com/bms/login/payload/response/UserInfoResponse.java

```diff
@@ -0,0 +1,49 @@
+package com.bms.login.payload.response;
+
+import java.util.List;
+
+public class UserInfoResponse {
+    private Long id;
+    private String username;
+    private String email;
+    private List<String> roles;
+
+    public UserInfoResponse(Long id, String username, String email, List<String> roles) {
+        this.id = id;
+        this.username = username;
+        this.email = email;
+        this.roles = roles;
+    }
+
```

Comment:
Getters & setters can be avoided if @Builder, @Value are used

Reply:
Okay